man/mkdocs/mkdocs.yml

```diff
@@ -23,19 +24,21 @@ theme:
   logo: grass_logo.svg
   favicon: favicon.ico
   features:
+    - content.action.edit # Edit on GitHub
+    - content.action.view
+    - content.code.copy
     - search.suggest
     - search.highlight
-    - content.code.copy
     - navigation.footer
     - navigation.indexes
-    # - navigation.expand
+    - navigation.instant
+    - navigation.instant.progress
     - navigation.tabs
     - navigation.tabs.sticky
-    - content.action.view
-    - content.action.edit # Edit on GitHub
+    - navigation.top
+
   palette:
     primary: custom
-    # accent: custom
```

Comment:
Clean up comments

Reply:
I removed that comment. Do you mean something else?